### DISCUSSION TITLE
Verification of Account

### DISCUSSION THREAD

**Discussion by SamKrasnoff** [community/community#118840] [GitHub Education]
### Hi everyone,

I have submitted both an ID and a hiring form as a method of verification that I am a teacher/researcher of the university I am at for Github Education. None have been approved, and the next steps are extremely unclear. I'd really appreciate some guidance from the community or GitHub employees for a process that should be fairly simple.

**Comment by drannmari** [comment]
I'm sorry I have no help other than I have the same problem. My "issue" was that my last name is De Mars and the university letter has a space in my name and my billing info did not, as some websites do not accept spaces or consider it two names. So, I changed my billing info to have the space and it is still not accepted.

**Comment by zied-snoussi** [comment]
Hi SamKrasnoff,

Getting verified for Github Education can be tricky, so don't worry! It sounds like your ID and hiring form weren't accepted. Here's how we can fix this:

* **Talk to Github Education directly.** They can explain why your documents weren't approved and what to do next. Find their contact info on the Github Education website.
* **Double check what documents are needed.** The Github Education website should have a section on verifying your identity as a teacher/researcher. This will list the accepted document types and how to format them. 
* **Try resubmitting your documents.** Once you know why they were rejected, fix the issue and submit them again.

**Important:** Don't scan your ID or form and translate them! These documents might have private info, so keep them secure.

Good luck getting your Github Education offer!

**Comment by samus-aran** [comment]
Hi @SamKrasnoff 

> I have submitted both an ID and a hiring form as a method of verification that I am a teacher/researcher of the university I am at for Github Education.

You are applying at a faculty member you need to provide faculty ID or evidence that you are faculty member. A student ID will not be sufficient as a teaching assistant. Aquire a suitable form of evidence as a faculty member. 👍
